server/server.go

```diff
@@ -57,7 +58,8 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 
 // Server wraps a HTTP and gRPC server, and some common initialization.
 //
-// Servers will be automatically instrumented for Prometheus metrics.
+// Servers will be automatically instrumented for Prometheus metrics
+// and Loki tracing.  HTTP over gRPC
```

Comment:
surely "whatever OpenTracing you have configured", not necessarily Loki.

Reply:
Sorry bad merge there I think; that shouldn't be there.